app/src/main/java/org/sopt/sample/presentation/main/fragment/HomeFragment.kt

```diff
@@ -20,7 +21,7 @@ class HomeFragment : Fragment() {
         get() = requireNotNull(_binding) { "홈 프래그먼트에서 _binding이 널임" }
     private val viewModel by viewModels<HomeViewModel>()
     private val adapter by lazy { FollowersAdapter(requireContext()) }
-    private var userList: List<ResponseGetUserDTO.User>? = listOf()
+    private var userList: List<ResponseGetUserDto.User>? = listOf()
```

Comment:
```suggestion
    private var userList: List<ResponseGetUserDto.User> = listOf()
```
`?`연산자를 넣어준 이유가 있을까여?

Reply:
만약 서버에서 널값이 올까봐,,였는데 그럴 일 없겠네?? 애초에 DTO도 널러블하지 않게 작성했구. 수정할게 !!